discretize/View.py

```diff
@@ -1011,6 +1018,7 @@ def plotImage(self, *args, **kwargs):
         return self._plotCylTensorMesh('plotImage', *args, **kwargs)
 
 
+@requires({'matplotlib': matplotlib})
```

Comment:
Thoughts on this going on the class not the plotting methods?

Reply:
I thought about this too. My reason to put it on the lower level was in case that the class is also needed for other stuff which I didn't want to break, and I didn't want to check everything... But I would be all in to move it up to the class level.

Reply:
At this point I don't have strong opinions either way, so I am fine leaving this as is, and later on we could move it up if that makes sense? 

This also could be nice down the line if we want to support other plotting backends (e.g. plotly)